aiogoogle/sessions/aiohttp_session.py

```diff
@@ -11,7 +11,12 @@
 
 from ..models import Response
 from .abc import AbstractSession
+import os
 
+try:
+    HTTP_PROXY = str(os.getenv("AIOGOOGLE_HTTP_PROXY", ""))
```

Comment:
change to: `HTTP_PROXY = os.getenv("AIOGOOGLE_HTTP_PROXY", "")`

Also, no need for try except